include/irr/asset/ICPUTextureView.h

```diff
@@ -10,6 +10,11 @@ namespace asset
 
 class ICPUTextureView : public IAsset, public IDescriptor
 {
+public:
+    size_t conservativeSizeEstimate() const override { return 0ull /*TODO*/; }
```

Comment:
the size is only the creation parameters (mip range, layer range, format, and handle to actual texture to create view from, have I missed anything?)

Reply:
@Crisspl ?

Reply:
to implement when it's actually getting done